content/tutorial-paychan.md

```diff
@@ -0,0 +1,489 @@
+# Payment Channels Tutorial
+
+Payment Channels are an advanced feature for sending "asynchronous" XRP payments that can be divided into very small increments and settled later. This tutorial walks through the entire process of using a payment channel, with examples using the [JSON-RPC API](reference-rippled.html) of a local `rippled` server.
+
+1. **The payer creates a payment channel to a particular recipient.** In doing so, the payer pre-funds the channel with XRP.
+
+    This is a [PaymentChannelCreate transaction][]. As part of this process, the payer sets certain specifics of the channel like an expiration time and a settlement delay, which affect the guarantees around the claims in the channel. The payer also sets the public key that will be used to verify claims against the channel.
```

Comment:
Respectfully, I disagree. This is _contextual_ information that reminds users what they are doing in this step.

Technically, you are correct that some clauses, like "which affect the guarantees around the claims in the channel," are conceptual information. However, short phrases connect the conceptual information to the steps of the tutorial, which I think is the point of an educational tutorial like this.